### DISCUSSION TITLE
How to display post-processing figures

### DISCUSSION THREAD

**Discussion by XingchunCat** [ansys/pyfluent#1571] [Q&A]
I was trying the [Mixing tube example ](https://fluent.docs.pyansys.com/release/0.13/examples/00-fluent/mixing_elbow_settings_api.html#) with Jupyterlab. After solving it suppose to create a velocity vector contour. I do observe the code execute successfully but there is no image displayed. Does anyone know how to display plots? Really appreciate if someone can help me out. Thanks in advance.

Additional Info:
Jupyterlab V3.5
Fluent 2022R1
has matplotlib installed

**Comment by mkundu1** [comment]
@XingchunCat Please use the [example](https://visualization.fluent.docs.pyansys.com/version/stable/examples/00-postprocessing/post_processing_exhaust_manifold.html#sphx-glr-examples-00-postprocessing-post-processing-exhaust-manifold-py) from the visualization library for the client-side graphics example.
